curriculum/challenges/english/25-front-end-development/quiz-git/66f1b06a5a5d10cc100af620.md

```diff
@@ -17,439 +17,438 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+Who can use `git`?
 
 #### --distractors--
 
-Placeholder distractor 1
+Developers
 
 ---
 
-Placeholder distractor 2
+Administrators
 
 ---
 
-Placeholder distractor 3
+Subscribers/Premium Users
 
 #### --answer--
 
-Placeholder answer
+Everyone
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Why is using version control software a best practice for high performing software and DevOps teams?
 
 #### --distractors--
 
-Placeholder distractor 1
+A complete long-term change history of every file
 
 ---
 
-Placeholder distractor 2
+Branching and merging
 
 ---
 
-Placeholder distractor 3
+Traceability
 
 #### --answer--
 
-Placeholder answer
+All of the options
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What are the main advantages of using sites like GitHub and GitLab?
 
 #### --distractors--
 
-Placeholder distractor 1
+They allow online shopping.
 
 ---
 
-Placeholder distractor 2
+They are a social media platform for posting photos.
 
 ---
 
-Placeholder distractor 3
+They only act as cloud storage for individual files.
 
 #### --answer--
 
-Placeholder answer
+They provide version control and collaboration for code development.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command would not display the current `git` version?
 
 #### --distractors--
 
-Placeholder distractor 1
+`git --version`
 
 ---
 
-Placeholder distractor 2
+`git -v`
 
 ---
 
-Placeholder distractor 3
+`git version`
 
 #### --answer--
 
-Placeholder answer
+`git v`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which statement is true regarding encryption?
 
 #### --distractors--
 
-Placeholder distractor 1
+SSH keys cannot be used for encrypting data and communication.
 
 ---
 
-Placeholder distractor 2
+Neither GPG keys nor SSH keys can be used for encrypting data and communication.
 
 ---
 
-Placeholder distractor 3
+Both GPG and SSH keys can be used for encrypting data and communication.
```

Comment:
You actually *can* use SSH keys to encrypt and decrypt communication. It's janky, and not at all preferred, but doable.